minio/minio_test.go

```diff
@@ -26,25 +26,24 @@ func Test_Get(t *testing.T) {
 	)
 
 	err := testStore.Set(key, val, 0)
-	utils.AssertEqual(t, nil, err)
+	require.NoError(t, err)
 
 	result, err := testStore.Get(key)
-	utils.AssertEqual(t, nil, err)
-	utils.AssertEqual(t, val, result)
+	require.NoError(t, err)
+	require.Equal(t, val, result)
 
 	result, err = testStore.Get("doe")
-	utils.AssertEqual(t, true, len(result) == 0)
```

Comment:
`result` is unused here. Should be check with `require.Zero(t, len(result))`

Reply:
done